src/api/ethereum.ts

```diff
@@ -215,16 +214,6 @@ export const startFlow = async (
 							maxPriorityFeePerGas,
 						},
 					}),
-					await framework.idaV1.approveSubscription({
-						superToken: SUSHIxAddress,
-						indexId: '2',
-						publisher: exchangeAddress,
-						userData,
-						overrides: {
-							maxFeePerGas,
-							maxPriorityFeePerGas,
-						},
-					}),
```

Comment:
Bring this back, change from SUSHIx to use ETHx for now since youre removing sushix. Ill bring back REX LP soon